### ISSUE TITLE
environment configuration

### ISSUE THREAD

**Issue by mikedh-avia** [iDonghq/FAST_LIO_LOCALIZATION_PLUS#3]
Thank you for sharing. Do you need the same environment configuration as fastlio?Is the architecture based on arm64 and melodic feasible?

**Comment by iDonghq** [comment]
The configuration should be slightly changed for adjusting to different architecture, as we all know the platform of embedded generally have weak computility.

**Comment by iDonghq** [comment]
> Thank you for sharing. Do you need the same environment configuration as fastlio?Is the architecture based on arm64 and melodic feasible?

The configuration should be slightly changed for adjusting to different architecture, as we all know the platform of embedded generally have weak computility.

**Comment by mikedh-avia** [comment]
> > Thank you for sharing. Do you need the same environment configuration as fastlio?Is the architecture based on arm64 and melodic feasible?
> 
> The configuration should be slightly changed for adjusting to different architecture, as we all know the platform of embedded generally have weak computility.

Thanks for the reply, I have tested fastlio2 on my nvidia and it works fine. Does this mean that this code will work on my computer as well?

**Comment by iDonghq** [comment]
> > > Thank you for sharing. Do you need the same environment configuration as fastlio?Is the architecture based on arm64 and melodic feasible?
> > 
> > 
> > The configuration should be slightly changed for adjusting to different architecture, as we all know the platform of embedded generally have weak computility.
> 
> Thanks for the reply, I have tested fastlio2 on my nvidia and it works fine. Does this mean that this code will work on my computer as well?

of course, cause Localization needs less computility than fastlio2 with mapping. There is no map-updating in the code.

**Comment by iDonghq** [comment]
> > > Thank you for sharing. Do you need the same environment configuration as fastlio?Is the architecture based on arm64 and melodic feasible?
> > 
> > 
> > The configuration should be slightly changed for adjusting to different architecture, as we all know the platform of embedded generally have weak computility.
> 
> Thanks for the reply, I have tested fastlio2 on my nvidia and it works fine. Does this mean that this code will work on my computer as well?

of course, cause Localization needs less computational complexities than fastlio2 with mapping. There is no map-updating in the code.
